.pre-commit-config.yaml

```
@@ -1,10 +1,13 @@
 repos:
   - repo: https://github.com/mineiros-io/pre-commit-hooks
-    rev: v0.3.0
+    rev: v0.3.1
     hooks:
       - id: terraform-fmt
       - id: terraform-validate
         exclude: ^examples|.terraform/
       - id: tflint
       - id: golangci-lint
       - id: phony-targets
+      - id: markdown-link-check
+        args: ['-p']   # When adding the -p flag, markdown-link-check will always with an exit code 0, even if dead links are found
+        verbose: true  # Forces the output of the hook to be printed even when the hook passes.
```

Indentation:

```suggestion
      - id: markdown-link-check
          args: ['-p']   # When adding the -p flag, markdown-link-check will always with an exit code 0, even if dead links are found
          verbose: true  # Forces the output of the hook to be printed even when the hook passes.
```

Also, since the check is for markdown links, what is the purpose of exiting with 0 when a dead link is found ? (I thought the idea would be to fail a CI if there is a broken link or something).

---

indentation is wrongly suggested, implicit keys need to be on a single line

The idea of having the CI pass is to show the developer that links are breaking but it shouldn't break the build (eg. sometimes that happens on purpose, eg. when a released hasn't been scheduled yet but the link was already implemented in the README.md to prepare the release).